packages/cli/api-importers/openapi/v2-importer-asyncapi/src/2.x/types.ts

```diff
@@ -0,0 +1,66 @@
+import { OpenAPIV3 } from "openapi-types";
```

Comment:
https://www.npmjs.com/package/json-schema-to-typescript is the json schema to typescript types generation -- would recommend using this and starting with the official asyncapi json schema